code/modules/atmospherics/machinery/portable/pump.dm

```diff
@@ -155,3 +155,9 @@
 				replace_tank(usr, FALSE)
 				. = TRUE
 	update_icon()
+
+/obj/machinery/portable_atmospherics/pump/CtrlShiftClick(mob/user)
```

Comment:
Why not alt click instead of complex ctl click

Reply:
Alt click takes out a tank